tools/verdict-back-ends/STEM/OwlModels/.gitignore

```diff
@@ -0,0 +1,6 @@
+/BaseModel.owl
+/Queries.owl
+/Run.owl
+/STEMRules.owl
+/SadlBuiltinFunctions.owl
+/SadlImplicitModel.owl
```

Comment:
What is this file supposed to do (that is, what do you want to tell git to do)?  Even though you've listed the OWL files in this file, all these OWL files are already in source control so git still will keep pulling or committing any changes to these files.  I don't think we can remove the OWL files either because the STEM Runner requires the OWL files and we don't have an automated batch SADL to OWL translator that can create the OWL files automatically.  Therefore, this file doesn't really have any effect on git so I want to ask what you wanted it to do.